src/Reflection/ReflectionParameter.php

```diff
@@ -317,7 +318,7 @@ public function getDefaultValue()
      */
     public function getDefaultValueAsString() : string
     {
-        return var_export($this->getDefaultValue(), true);
+        return ValueDumper::dump($this->getDefaultValue());
```

Comment:
`ValueDumper` is not needed here: `var_export()` is portable, stable, security-checked and does exactly what is needed here